gradle.properties

```diff
@@ -1,11 +1,12 @@
-org.gradle.daemon=false
-
-mcversion=1.15.2
-forgeversion=31.1.43
-forgegroup=net.minecraftforge
-mcp_mappings=20200411-1.15.1
-curse_project_id=238222
-
-version_major=6
-version_minor=0
-version_patch=1
+org.gradle.daemon=false
+
+mcversion=1.15.2
+forgeversion=31.1.43
+forgegroup=net.minecraftforge
+mcp_mappings=20200411-1.15.1
+curse_project_id=238222
+
+version_major=6
+version_minor=0
+version_patch=2
+
```

Comment:
For some reason Github says this whole file changed...?

Reply:
I can clean this up myself and merge, one sec

Reply:
Ugh. Yeah, I didn't realize there was another JEI update today. And then I used GitHub to resolve the conflict instead of merging from the IDE. Didn't think that would be an issue.